tasks/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
+- name: Configure repositories
```

Comment:
As the above note, I think a better block name could be: "Configure epel repository" instead of "Configure repositories"